src/Directory.Build.props

```diff
@@ -3,11 +3,10 @@
 <Project>
 
   <PropertyGroup>
-    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileDirectory)..\Directory.Build.props</MSBuildAllProjects>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
```

Comment:
This is incorrect, Directory.Build.props are added to MSBuildAllProjects by the file that imports them, they do not add themselves.

Reply:
I'm not sure I follow why that would be the case. Shouldn't every file be adding itself?

---
In reply to: [184184946](https://github.com/dotnet/sdk/pull/2178#discussion_r184184946) [](ancestors = 184184946)

Reply:
I see. Will fix. Thanks for pointing it out!

---
In reply to: [184193805](https://github.com/dotnet/sdk/pull/2178#discussion_r184193805) [](ancestors = 184193805)